RELEASE/scripts/autoscend/auto_bedtime.ash

```diff
@@ -1366,6 +1366,11 @@ boolean doBedtime()
 			auto_log_info("You have a tea tree to shake!", "blue");
 		}
 
+		if((item_amount($item[August Scepter]) > 0) && (get_property("_augSkillsCast").to_int() < 5))
```

Comment:
For iotm, we make special functions to know of we have it or not. Use `auto_haveAugustScepter()` instead of `item_amount`